### DISCUSSION TITLE
Struggling to get a right callback URL

### DISCUSSION THREAD

**Discussion by RodiCoci** [etsy/open-api#1041] [Q&A] [answered]
Hi, 

I'm trying to make an personal app to upload listings on etsy. 

When I execute the following code I get "Authorization request successful. User should be redirected to the authorization page." a print in my terminal. But If I visit the URL I get from this code ("authorization_request_url") then I get on the error that "the requested redirect URL is not permitted."

Probably there is something wrong with my callback URL. When I applied for the app in the Etsy developpers section, I just made a quick website through 'Wix' to have an URL. I tried to change the "website URL*"-field in my app section in Etsy, but I got the error that "Callback URL does not match Application Domain."

Can somebody please help me to get a right callback URL and explain me what to take into account when making the right callback URL? 

![Code Github Question](https://github.com/etsy/open-api/assets/137859036/cb9cf716-4c2f-406a-8ec2-8850318cf522)
![Code Github Question Error](https://github.com/etsy/open-api/assets/137859036/eced8df2-9114-4b70-9b0f-d9d1b6491872)

**Comment by RodiCoci** [comment, marked as answer]
I found the solution yet. I had to fill in the URL again in this section. When I did this I got my access_token.
![Code Github Question Solution](https://github.com/etsy/open-api/assets/137859036/8f9aa791-d80a-47b5-ba12-0abe2f0a7909)

**Comment by gjaon** [comment]
In my case I don't see the edit your callback Url. What do I need to do to?
![etsyurl](https://github.com/etsy/open-api/assets/102289242/280e0304-add9-42ef-ac81-8e631d4c25aa)
